.github/workflows/mega-linter.yml

```diff
@@ -0,0 +1,122 @@
+          path: |
+            megalinter-reports
+            mega-linter.log
+
+      # Create pull request if applicable (for now works only on PR from same repository, not from forks)
+      - name: Create Pull Request with applied fixes
+        id: cpr
+        if: steps.ml.outputs.has_updated_sources == 1 &&
+            (env.APPLY_FIXES_EVENT == 'all' || env.APPLY_FIXES_EVENT ==
+            github.event_name) && env.APPLY_FIXES_MODE == 'pull_request' &&
+            (github.event_name == 'push' ||
+            github.event.pull_request.head.repo.full_name == github.repository) &&
+            !contains(github.event.head_commit.message, 'skip fix')
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ secrets.PAT || secrets.GITHUB_TOKEN }}
+          commit-message: "[MegaLinter] Apply linters automatic fixes"
+          title: "[MegaLinter] Apply linters automatic fixes"
+          labels: bot
+      - name: Create PR output
+        if: steps.ml.outputs.has_updated_sources == 1 &&
+            (env.APPLY_FIXES_EVENT == 'all' || env.APPLY_FIXES_EVENT ==
+            github.event_name) && env.APPLY_FIXES_MODE == 'pull_request' &&
+            (github.event_name == 'push' ||
+            github.event.pull_request.head.repo.full_name == github.repository) &&
+            !contains(github.event.head_commit.message, 'skip fix')
+        run: |
+          echo "Pull Request Number - ${{ steps.cpr.outputs.pull-request-number }}"
+          echo "Pull Request URL - ${{ steps.cpr.outputs.pull-request-url }}"
+
+      # Push new commit if applicable (for now works only on PR from same repository, not from forks)
+      - name: Prepare commit
+        if: steps.ml.outputs.has_updated_sources == 1 &&
+            (env.APPLY_FIXES_EVENT == 'all' || env.APPLY_FIXES_EVENT ==
+            github.event_name) && env.APPLY_FIXES_MODE == 'commit' && github.ref
+            != 'refs/heads/main' && (github.event_name == 'push' ||
+            github.event.pull_request.head.repo.full_name == github.repository) &&
+            !contains(github.event.head_commit.message, 'skip fix')
+        run: sudo chown -Rc $UID .git/
+      - name: Commit and push applied linter fixes
+        if: steps.ml.outputs.has_updated_sources == 1 &&
+            (env.APPLY_FIXES_EVENT == 'all' || env.APPLY_FIXES_EVENT ==
+            github.event_name) && env.APPLY_FIXES_MODE == 'commit' && github.ref
+            != 'refs/heads/main' && (github.event_name == 'push' ||
+            github.event.pull_request.head.repo.full_name == github.repository) &&
+            !contains(github.event.head_commit.message, 'skip fix')
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          branch: ${{ github.event.pull_request.head.ref || github.head_ref || github.ref }}
+          commit_message: "[MegaLinter] Apply linters fixes"
+          commit_user_name: megalinter-bot
+          commit_user_email: nicolas.vuillamy@ox.security
```

Comment:
Should we use this email here?

Reply:
I guess not. Maybe we could reuse/revive https://github.com/egibot for this ?

Reply:
I've pushed a commit to address this, please have a look.